playbooks/roles/ocp-config/tasks/main.yaml

```diff
@@ -56,6 +56,14 @@
     args:
       chdir: "{{ workdir }}"
 
+  - name: Generate Machine Config Master and Worker nodes
+    when:
+      - luks.enabled and luks is defined
```

Comment:
`luks is defined` should come before checking `luks.enabled` otherwise it will fail when luks is not defined.